src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Blocks/categories.html.twig

```diff
@@ -0,0 +1,69 @@
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+<div class="form-group mb-4" id="categories">
+  <h2>Categories <span class="help-box" data-toggle="popover" data-content="Where should the product be available on your site? The main category is where the product appears by default: this is the category which is seen in the product page's URL. Disabled categories are written in italics." data-original-title="" title=""></span></h2>
+
+  <div class="categories-tree js-categories-tree">
+    <fieldset class="form-group hide">
+      <div class="ui-widget">
+        <div class="search search-with-icon">
+          <input type="text" id="ps-select-product-category" class="form-control autocomplete search mb-1 ui-autocomplete-input" placeholder="Search categories" autocomplete="off">
+        </div>
+
+        <label class="form-control-label text-uppercase">Associated categories</label>
+
+        <div id="ps_categoryTags" class="pstaggerTagsWrapper" style="display: block;">
+          <span class="pstaggerTag"><span data-id="2" title="Root > Home">Home</span><a class="pstaggerClosingCross" href="#" data-id="2">x</a></span>
+        </div>
+
+        <div id="ps_categoryTree" class="hide">
+          [{"id":"1","name":"Root","breadcrumb":"Root", "childs": [
+            {"id":"2","name":"Home","breadcrumb":"Root &gt; Home", "childs": [
+              {"id":"6","name":"Accessories","breadcrumb":"Root &gt; Home &gt; Accessories", "childs": [
+              {"id":"7","name":"Stationery","breadcrumb":"Root &gt; Home &gt; Accessories &gt; Stationery"}
+              ]
+              },
+              {"id":"3","name":"Clothes","breadcrumb":"Root &gt; Home &gt; Clothes", "childs": [
+                {"id":"4","name":"Men","breadcrumb":"Root &gt; Home &gt; Clothes &gt; Men"},{"id":"5","name":"Women","breadcrumb":"Root &gt; Home &gt; Clothes &gt; Women"}
+              ]}
+            ]
+            }]}]
+        </div>
+
+        <div class="categories-tree-actions js-categories-tree-actions">
+          <span class="form-control-label" id="categories-tree-expand" style=""><i class="material-icons">expand_more</i>Expand</span>
+          <span class="form-control-label hide" id="categories-tree-reduce"><i class="material-icons">expand_less</i>Collapse</span>
+        </div>
+
+        <div id="form_step1_categories" class="category-tree-overflow js-category-tree-overflow"></div>
+      </div>
+    </fieldset>
+
+    <div class="categories-tree-loader">
+      <div class="spinner m-auto d-block"></div>
+    </div>
+  </div>
+</div>
```

Comment:
Don't review this for the moment, it will change (ps_categoryTree will be kicked and translations will be added)